.gitignore

```diff
@@ -0,0 +1,4 @@
+.env
+build/
+node_modules/
+coverage/
```

Comment:
I would put `/build` and `/coverage`inside `.gitignore` so it wouldn't be checked in into the repository. I

 If the build folder is checked in it will cause duplication or unnecessary code and the duplication can increase the repository size.

I saw that you have that on line 2 but the path might be wrong since I can see the build folder in the repository.